config/locales/en.yml

```diff
@@ -315,6 +315,11 @@ en:
       popular_links_title: Most read
       related_links_title: Related guides
 
+    inline_sign_up:
```

Comment:
Maybe we should group this `inline_sign_up` with [`newsletter_subscriptions`](https://github.com/moneyadviceservice/frontend/blob/7233-signup-article-end-copy/config/locales/en.yml#L385) and rename it to `inline_signup`?

_Side Note_: There is also a [`signup_inpage`](https://github.com/moneyadviceservice/frontend/blob/7233-signup-article-end-copy/config/locales/en.yml#L406) with similar content which is a bit of a smell :dash: